modules/mcagar/src/main/ts/ephox/mcagar/api/TinyHooks.ts

```diff
@@ -14,19 +14,25 @@ export interface ShadowRootHook<T extends EditorType> extends Hook<T> {
   readonly shadowRoot: () => SugarElement<ShadowRoot>;
 }
 
+export interface SetupElement {
```

Comment:
Note for review: New way to place an inline editor on a specific element (and tear it down). See implementation here: https://github.com/tinymce/tinymce/pull/6418/files#diff-6c77c3ab22fb82e21ed2a5d60517083bae429305c54c5f79768470002472a680R11